android/src/main/java/com/tailscale/ipn/ui/viewModel/LoginQRViewModel.kt

```diff
@@ -20,12 +20,28 @@ import kotlinx.coroutines.launch
 
 class LoginQRViewModel : IpnViewModel() {
 
+  val numCode: StateFlow<String?> = MutableStateFlow(null)
   val qrCode: StateFlow<ImageBitmap?> = MutableStateFlow(null)
 
   init {
     viewModelScope.launch {
       Notifier.browseToURL.collect { url ->
-        url?.let { qrCode.set(generateQRCode(url, 200, 0)) } ?: run { qrCode.set(null) }
+        url?.let {
+          qrCode.set(generateQRCode(url, 200, 0))
+          // Extract the string after "https://login.tailscale.com/a/"
+          val prefix = "https://login.tailscale.com/a/"
```

Comment:
This is not going to work with a custom coordination server.

Reply:
If there's a custom server, then we aren't showing the code